spec/log_in_spec.rb

```diff
@@ -0,0 +1,21 @@
+feature 'Logging in to Chitter' do
+  let!(:user) do
+    User.create(name: 'Katie', username: 'KS', email: 'K@email.com', password: 'secret', password_confirmation: 'secret')
```

Comment:
Metrics/LineLength: Line is too long. [121/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)